plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/hooks.ts

```diff
@@ -128,6 +128,13 @@ export default ({
       name: fieldName["buttonLink"],
       onClick: handleFieldAdd({}),
     },
+    story: {
+      name: fieldName["story"],
+      onClick: handleFieldAdd([
+        { title: "掛川城を外から眺める", id: "1" },
+        { title: "掛川城の内部を見てみる", id: "2" },
```

Comment:
These two stories are boilerplate. Please remove them.
Also, handleFieldAdd accepts an object, not an array. If you were going to pass in default stories, it should be
```
{
  stories: [{id: generateID(), title: "asldjfkasdf"}]
}
```
But please just make it `{}`

Reply:
done

Reply:
9f9de75f0a1fd71f52a177c41eaf8ccb46457fd5

Reply:
Sorry maybe my message wasn't clear. Please just make it an empty object

Reply:
done